### DISCUSSION TITLE
Translation result as a note / Note feature

### DISCUSSION THREAD

**Discussion by davidvelosa** [koreader/koreader#7268] [Ideas]
Hi Koreader team! First and foremost thanks for the amazing app and work!!

I was thinking that it would be very useful to all of us who use our readers as a language learning tool (and studying in general) to integrate the translations and the notes somehow, so I have these two ideas.

1. Add an option in the translate box, either for:
- Copying the translation result text (to add it later as a highlight note), or
- For adding the translation result into a note text for the same highlighted text

2. Also to see the note as soon as you tap in the highlighted text, in a non intrusive way. (or to have an option to activate this type of visualization on the settings)

Thanks in advance, and keep the good work

**Comment by Frenzie** [comment]
I'm afraid it'll probably be a PRs welcome kind of situation, but at least some of those should be relatively simple. :-)

**Comment by poire-z** [comment]
Might be dependant on reworking the mess our highlights/notes is, cf #5794.
And being able to associate multiple notes/external info to a highlight: translations is nice, but one may wants to add a dictionary result, or wikipedia summary.
